psi4/src/read_options.cc

```diff
@@ -178,7 +178,7 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
     /*- What algorithm to use for the SCF computation. See Table :ref:`SCF
     Convergence & Algorithm <table:conv_scf>` for default algorithm for
     different calculation types. -*/
-    options.add_str("SCF_TYPE", "PK", "DIRECT DF MEM_DF DISK_DF PK OUT_OF_CORE CD GTFOCK");
+    options.add_str("SCF_TYPE", "PK", "DIRECT DF MEM_DF DISK_DF PK OUT_OF_CORE CD GTFOCK COSX");
```

Comment:
should this be `DFJ_COSX` for future composite compatibility? or does `COSX` imply the DFJ widely enough?

Reply:
The latter, `DFJ` is the only sensible J algorithm to be combined with `COSX`, so I left it off the `SCF_TYPE` keyword in order to be more user-friendly. The `JK` class is given the full name `DFJCOSK` because it isn't user-facing